telemetry/definitions/commonDefinitions.json

```diff
@@ -2210,13 +2210,15 @@
                     "required": false
                 },
                 {
-                    "type": "cwsprChatFullDisplayLatency"
+                    "type": "cwsprChatFullDisplayLatency",
+                    "required": false
```

Comment:
Just to call it out for anyone else looking at this: these will need to be implemented when we moved to flare but they aren't really trivial to implement